op-chain-ops/genesis/config.go

```diff
@@ -216,14 +216,15 @@ func (d *DeployConfig) Check() error {
 	if d.L2GenesisBlockBaseFeePerGas == nil {
 		return fmt.Errorf("%w: L2 genesis block base fee per gas cannot be nil", ErrInvalidDeployConfig)
 	}
```

Comment:
My preference would be to add a bool that is enable/disable gov token instead of making the config implicit on each other, the other config values only apply if the bool value is set

Reply:
done